<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardCreateDialog</class>
 <widget class="QDialog" name="StandardCreateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="dialogContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>674</width>
       <height>518</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>674</width>
       <height>518</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#dialogContainer {
	background-color: white;
	border-radius: 12px;
	border: none;
}

#bodyContainer {
	border-bottom-left-radius: 12px;
	border-bottom-right-radius: 12px;
}


QLabel {
	color: #1A1A1A;
	font-family: Open Sans;
	font-weight: 600;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
}

QLineEdit {
	color: #1A1A1A;
	background: white;
	font-family: Open Sans;
	font-weight: 400;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
	border: 1px solid #CCCCCC;
	padding-left: 12px;
	padding-right: 12px;
}

QTextEdit {
	color: #1A1A1A;
	background: white;
	font-family: Open Sans;
	font-weight: 400;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
	border: 1px solid #CCCCCC;
	padding-left: 12px;
	padding-right: 12px;
}

QComboBox {
    background-color: white;
    border: 1px solid #ccc;
    border-radius: 8px;
    padding: 4px;
    font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
    color: #333;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border: none;
}

QComboBox::down-arrow {
    image: url(:/images/chevron-down.svg);
    width: 16px;
    height: 16px;
}

QComboBox::down-arrow:on {
    image: url(:/images/chevron-up.svg);
}

QComboBox QAbstractItemView {
    background-color: white;
    margin-top: 0px;
    border: 1px solid #ccc;
    border-radius: 8px;
	outline: none;
}

/* Each dropdown item */
QComboBox QAbstractItemView::item {
	color: #1A1A1A;
    padding: 6px 12px;
    border-radius: 4px;
}

/* Hovered item */
QComboBox QAbstractItemView::item:hover {
    background-color: #e6f2ff;
}

/* Selected item */
QComboBox QAbstractItemView::item:selected {
    background-color: #d0e8ff;
    color: #1a1a1a;
}

#cancelButton {
	color: #1A1A1A;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 18px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
	border: 1.5px solid #1A1A1A
}

#saveButton {
    background-color: #FBAE17;
	color: white;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 18px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="titleBar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>18</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>16</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>16</number>
         </property>
         <item>
          <widget class="QLabel" name="dialogTitle">
           <property name="styleSheet">
            <string notr="true">font-family: Open Sans;
font-size: 22px;
font-weight: 500;
color: black;
line-height: 30px;
letter-spacing: 0%;
</string>
           </property>
           <property name="text">
            <string>Standard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="closeButton">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../../Resources/Resources.qrc">
             <normaloff>:/images/Project/cross-icon.svg</normaloff>:/images/Project/cross-icon.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="bodyContainer" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>18</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>18</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>18</number>
         </property>
         <item>
          <widget class="QWidget" name="TitleContainer" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>68</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="titleLabel">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="titleEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="UploadContainer" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="apiKeyLabel">
              <property name="text">
               <string>Upload Standard</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="DropFileWidget" name="uploadFileWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>638</width>
                <height>244</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>638</width>
                <height>244</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="buttonContainer" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>42</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="buttonLayout">
            <property name="spacing">
             <number>18</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="cancelButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>42</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>42</height>
               </size>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>42</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>42</height>
               </size>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropFileWidget</class>
   <extends>QWidget</extends>
   <header location="global">dropfilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
